### ISSUE TITLE
Wordpress 4.6.1 download fail

### ISSUE THREAD

**Issue by avvertix** [avvertix/wp-unittest-tools#1]
php 7.0

```
1) TestConfigureCommand::testCommandExecution
RuntimeException: Unable to open ./tmp/wordpress/includes/factory/ using mode w+: fopen(./tmp/wordpress/includes/factory/): failed to open stream: Invalid argument

C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\psr7\src\functions.php:299
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\psr7\src\functions.php:307
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\psr7\src\LazyOpenStream.php:37
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\psr7\src\StreamDecoratorTrait.php:31
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\psr7\src\StreamDecoratorTrait.php:136
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\Handler\CurlFactory.php:366
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\Handler\CurlHandler.php:40
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\Handler\Proxy.php:28
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\Handler\Proxy.php:51
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\PrepareBodyMiddleware.php:42
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\Middleware.php:30
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\RedirectMiddleware.php:68
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\Middleware.php:59
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\HandlerStack.php:67
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\Client.php:275
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\Client.php:123
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\guzzlehttp\guzzle\src\Client.php:129
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\src\Traits\InteractWithHttp.php:27
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\src\Commands\InstallWordpressTestSuiteCommand.php:84
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\symfony\console\Command\Command.php:257
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\src\Traits\InteractWithCommands.php:21
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\src\Commands\ConfigureCommand.php:94
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\symfony\console\Command\Command.php:257
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\vendor\symfony\console\Tester\CommandTester.php:80
C:\Users\Alessio\Documents\GitHub\wp-unittest-tools\tests\TestConfigureCommand.php:24
```

**Comment by avvertix** [comment]
caused by a folder named `factory` which was added in Wordpress 4.5 https://develop.svn.wordpress.org/tags/4.6.1/tests/phpunit/includes/

**Comment by avvertix** [comment]
closed by commit 900cddf3234d1418d532adeef095471e172ef6a2
